scripts/control_bar.js

```diff
@@ -32,6 +32,9 @@ var makeImageControl = function(content) {
 
     $(image).mousedown( function(event) {
```

Comment:
would it be possible to set the mousedown on the content[0] div containing the img? That way we don't need to explicitly set focus as it would be on that component perhaps? Maybe not.

Reply:
Bear in mind that I'm a total ignoramus as far as js goes -- I'm not entirely sure what this means? Does this mean default to image having focus when the window opens, or something else, entirely?

Reply:
I'll try it.

Reply:
I tried Chris's suggestion which I believe is just intended to simplify the code without changing the behaviour. The cursor drag works but the line `content[0].focus();` is still needed to make the arrow keys work. I don't understand why, peraps something to do with tabindex.

Reply:
worth a shot. thanks for trying it out.